e2e/cypress/integration/channel/channel_settings_spec.js

```diff
@@ -52,4 +54,32 @@ describe('Channel Settings', () => {
             cy.get('@channel').find('span.btn-close').should('exist');
         });
     });
+
+    it('MM-T882 Channel settings / Channel URL validation works properly', () => {
+        // # Login as admin and create new channel
+        cy.apiAdminLogin();
+        cy.apiCreateChannel(testTeam.id, 'coolest-channel', 'coolest channel').then(({channel}) => {
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+            // # Go to channel dropdown > Rename channel
+            cy.get('#channelHeaderDropdownIcon').click();
+            cy.findByText('Rename Channel').click();
+
+            // # Try to enter existing URL and save
+            cy.get('#channel_name').clear().type('town-square');
+            cy.get('#save-button').click();
+
+            // # Error is displayed and URL is unchanged
+            // expect('.control-label').to.be.visible.and.to.eq('');
```

Comment:
Test case updated with correct error message. Then this can be,
```suggestion
            cy.get('.has-error').should('be.visible').and('contain', 'Unable to update the channel.');
```

Reply:
I still don't see the verification for "On 3, an error is shown as "Unable to update the channel.""